tailwind.config.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
       colors: {
         primary: "#FFFFFF",
         secondary: "#000000",
-
+        gray:"#E5E7EB",
+        id_gray:"#797C7B",
```

Comment:
check this colors already mention in develop baranch then use it

Reply:
ok